docs/pages/versions/unversioned/sdk/auth-session.md

```diff
@@ -143,6 +149,65 @@ Cancels an active `AuthSession` if there is one. No return value, but if there i
 
 Get the URL that your authentication provider needs to redirect to. For example: `https://auth.expo.io/@your-username/your-app-slug`.
 
+> **Note** This method will throw an exception if you're using the bare workflow.
+
+## Usage in the bare React Native app
+
+In managed apps, `AuthSession` uses expo servers to create a proxy between your application and the auth provider. Unfortunately, you don't have access to these servers in bare apps. To overcome this problem, you can create your proxy service.
```

Comment:
It's better to say that we don't provide support for something rather than that you don't have access to something - the later has more negative meaning.
```suggestion
In managed apps, `AuthSession` uses Expo servers to create a proxy between your application and the auth provider. Unfortunately, we don't provide support to use these servers in bare apps. To overcome this, you can create your proxy service.
```